Modules/Role/Http/Services/RoleService.php

```diff
@@ -178,14 +198,14 @@ public function showAdmin(): array
      */
     public function getRole(): array
     {
-        $role = Role::where('role', '!=', 'Super Admin')
+        $role = Role::where('name', '!=', Role::ROLE_SUPER)
             ->get();
         $allRole = [];
         if (!empty($role)) {
             foreach ($role as $v) {
                 $allRole[] = array(
                     'id' => $v->id,
-                    'role' => $v->role
+                    'role' => $v->name
                 );
```

Comment:
Use proper names, instead of role it should be $roles and instead of $v use $role.

Reply:
Fixed